build/patches/Remove-google-fonts-from-gms.patch

```diff
@@ -0,0 +1,55 @@
+From: uazo <uazo@users.noreply.github.com>
+Date: Wed, 19 Jan 2022 12:39:24 +0000
+Subject: Remove fetching google fonts from gms
```

Comment:
Is this a defensive or a practical patch? e.g. did you add it because it is currently being triggered or just in case to avoid any future activation?

Reply:
it is practical.
the code is called [here](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/platform/fonts/android/font_unique_name_lookup_android.cc;l=121;bpv=1;bpt=1?q=FetchAllFontFiles&ss=chromium%2Fchromium%2Fsrc&start=41
) and here is also the [flag to deactivate](https://source.chromium.org/chromium/chromium/src/+/main:content/public/common/content_features.cc;l=36?q=kAndroidDownloadableFontsMatching&sq=&ss=chromium%2Fchromium%2Fsrc) but I preferred to remove only the call to gms.
what do you think?

Reply:
among other things that code is a nice example of calling mojom directly to java without going through the native... it can be useful.

Reply:
I am way behind in terms of mojom usage, but we should probably bookmark these useful snippets.

> it is practical.

Ok; shall we start adding this information in the patch descriptions?

> the code is called [here](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/platform/fonts/android/font_unique_name_lookup_android.cc;l=121;bpv=1;bpt=1?q=FetchAllFontFiles&ss=chromium%2Fchromium%2Fsrc&start=41) and here is also the [flag to deactivate](https://source.chromium.org/chromium/chromium/src/+/main:content/public/common/content_features.cc;l=36?q=kAndroidDownloadableFontsMatching&sq=&ss=chromium%2Fchromium%2Fsrc) but I preferred to remove only the call to gms.
> what do you think?

In this case then better to also toggle the flag, even if it does nothing when enabled? At least the disabled flag is in the "truthful" state and it might serve as a canary (breaking the patch) for when feature changes.